cli/internal/lockfile/pnpm_lockfile.go

```diff
@@ -327,11 +326,12 @@ func pruneImporters(importers map[string]ProjectSnapshot, workspacePackages []tu
 		workspace := workspacePath.ToUnixPath().ToString()
 		importer, ok := importers[workspace]
 
-		if !ok {
-			return nil, fmt.Errorf("Unable to find import entry for workspace package %s", workspace)
+		// If a workspace has no dependencies *and* it is only depended on by the
```

Comment:
Is this the only situation where a workspace won't show up as an importer?

Reply:
It's the only time I've seen a workspace omitted from the importers. If there's another scenario where `pnpm` omits a workspace, then this change will catch that as well.